input4MIPs_source_id_zn_1.json

```diff
@@ -0,0 +1,15 @@
+{
+    "source_id":{
+        "CR-CMIP-0-2-0":{
```

Comment:
If we went with this option, the idea would be that none of the ESGF keys would go in this file, they'd all be handled in a different file (which would link up via the source_id).